packages/yoga/src/ThemeProvider/themes/BaseTheme.js

```diff
@@ -254,6 +256,53 @@ const BaseTheme = ({ primary, secondary }) => {
         color: colors.gray[2],
       },
     },
+    checkbox: {
+      width: spacing.xlarge,
+      height: spacing.xlarge,
+      border: {
+        width: borders.medium,
+        color: colors.primary[3],
+        radii: 4,
```

Comment:
?

Reply:
We don't have a radii value `4` in our token, we just have `0`, `8` and `999`.
What do you suggest? Add radii `4` in our token file?

Reply:
Another option is to use checkbox `border-radius: 4px` direct in the styled component without using token/theme variable.

Reply:
You should talk to the design guy and ask him about it.

Reply:
We have talked with the DesignOps team and we decided to add a new `radii` token: `semiRouded`